.github/workflows/test.yaml

```diff
@@ -106,6 +106,40 @@ jobs:
         name: coverage.unit.out
         path: coverage.unit.out
 
+  integration-tests-enterprise-postgres:
+    environment: "Configure ci"
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.16'
+
+    - name: cache go modules
+      uses: actions/cache@v2.1.6
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+
+    - name: checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: run integration tests
+      run: make test.integration.enterprise.postgres
+      env:
+        KONG_ENTERPRISE_LICENSE: ${{ secrets.KONG_ENTERPRISE_LICENSE }}
+
+    - name: collect test coverage
+      uses: actions/upload-artifact@v2.2.4
+      with:
+        name: coverage.enterprisepostgres.out
+        path: coverage.enterprisepostgres.out
```

Comment:
We're collecting this coverage output but it doesn't look like we're using it. We need to add it to the coverage job.

Reply:
https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R263